src/coverlet.console/Program.cs

```diff
@@ -156,15 +156,15 @@ static int Main(string[] args)
                 var summary = new CoverageSummary();
                 int numModules = result.Modules.Count;
 
-                var totalLinePercent = summary.CalculateLineCoverage(result.Modules).Percent * 100;
-                var totalBranchPercent = summary.CalculateBranchCoverage(result.Modules).Percent * 100;
-                var totalMethodPercent = summary.CalculateMethodCoverage(result.Modules).Percent * 100;
+                var totalLinePercent = summary.CalculateLineCoverage(result.Modules).GetCoveragePercentage();
```

Comment:
This can just be `var totalLinePercent = summary.CalculateLineCoverage(result.Modules).Percent` if you update the `Percent` property to contain the necessary calculations. Applies everywhere this new method is used